addons/l10n_fr/data/template/account.account-fr.csv

```diff
@@ -472,8 +472,8 @@
 "pcg_6063","Maintenance and minor equipment supplies","606300","expense","account.account_tag_operating","False","Fournitures d'entretien et de petit équipement"
 "pcg_6064","Administrative supplies","606400","expense","account.account_tag_operating","False","Fournitures administratives"
 "pcg_6068","Other materials and supplies","606800","expense","account.account_tag_operating","False","Achats autres matières et fournitures"
-"pcg_6071","Goods for resale (or group) A","607100","expense","account.account_tag_operating","False","Ventes de marchandises (ou groupe) A"
-"pcg_6072","Goods for resale (or group) B","607200","expense","account.account_tag_operating","False","Ventes de marchandises (ou groupe) B"
+"pcg_6071","Purchase of goods (or group) A","607100","expense","account.account_tag_operating","False","Achat de marchandises (ou groupe) A"
```

Comment:
Should'nt it be "Achats de marchandises" ?
https://www.l-expert-comptable.com/plan-comptable/compte-607-achats-de-marchandises

Reply:
Sorry, my bad. It is edited now

Reply:
np, thanks!